client/lib/plans/constants.js

```diff
@@ -182,229 +396,71 @@ export const PLANS_LIST = {
-		},
+	},
+
+	[ PLAN_PERSONAL_2_YEARS ]: {
+		...getPlanPersonalDetails(),
+		availableFor: plan => includes( [ PLAN_FREE, PLAN_PERSONAL ], plan ),
+		getProductId: () => 1029,
+		getStoreSlug: () => PLAN_PERSONAL_2_YEARS,
+		getPathSlug: () => 'personal-2-years',
 	},
 
 	[ PLAN_PREMIUM ]: {
-		getTitle: () => i18n.translate( 'Premium' ),
-		getAudience: () => i18n.translate( 'Best for entrepreneurs' ),
-		getBlogAudience: () => i18n.translate( 'Best for professionals' ),
-		getPortfolioAudience: () => i18n.translate( 'Best for freelancers' ),
-		getStoreAudience: () => i18n.translate( 'Best for freelancers' ),
-		getPriceTitle: () => i18n.translate( '$99 per year' ), //TODO: DO NOT USE
+		...getPlanPremiumDetails(),
+		availableFor: plan => includes( [ PLAN_FREE, PLAN_PERSONAL, PLAN_PERSONAL_2_YEARS ], plan ),
 		getProductId: () => 1003,
-		getPathSlug: () => 'premium',
 		getStoreSlug: () => PLAN_PREMIUM,
-		availableFor: plan => includes( [ PLAN_FREE, PLAN_PERSONAL ], plan ),
-		getDescription: () =>
-			i18n.translate(
-				'{{strong}}Best for Entrepreneurs & Freelancers:{{/strong}}' +
-					' Build a unique website with advanced design tools, CSS editing, lots of space for audio and video,' +
-					' and the ability to monetize your site with ads.',
-				{
-					components: {
-						strong: (
-							<strong className="plans__features plan-features__targeted-description-heading" />
-						),
-					},
-				}
-			),
-		getFeatures: () =>
-			compact( [
-				// pay attention to ordering, shared features should align on /plan page
-				FEATURE_CUSTOM_DOMAIN,
-				FEATURE_JETPACK_ESSENTIAL,
-				FEATURE_EMAIL_LIVE_CHAT_SUPPORT,
-				FEATURE_UNLIMITED_PREMIUM_THEMES,
-				FEATURE_ADVANCED_DESIGN,
-				FEATURE_13GB_STORAGE,
-				FEATURE_NO_ADS,
-				isEnabled( 'republicize' ) && FEATURE_REPUBLICIZE,
-				isEnabled( 'simple-payments' ) && FEATURE_SIMPLE_PAYMENTS,
-				FEATURE_WORDADS_INSTANT,
-				FEATURE_VIDEO_UPLOADS,
-			] ),
-		getPromotedFeatures: () => [
-			FEATURE_CUSTOM_DOMAIN,
-			FEATURE_NO_ADS,
-			FEATURE_ADVANCED_DESIGN,
-			FEATURE_13GB_STORAGE,
-		],
-		getSignupFeatures: () =>
-			compact( [
-				FEATURE_ADVANCED_CUSTOMIZATION,
-				FEATURE_PREMIUM_THEMES,
-				FEATURE_ALL_PERSONAL_FEATURES,
-			] ),
-		getBlogSignupFeatures: () => [
-			FEATURE_MONETISE,
-			FEATURE_PREMIUM_THEMES,
-			FEATURE_ALL_PERSONAL_FEATURES,
-		],
-		getPortfolioSignupFeatures: () => [
-			FEATURE_ADVANCED_CUSTOMIZATION,
-			FEATURE_PREMIUM_THEMES,
-			FEATURE_ALL_PERSONAL_FEATURES,
-		],
-		getBillingTimeFrame: abtest => {
-			if ( abtest && abtest( 'upgradePricingDisplayV2' ) === 'modified' ) {
-				// Note: Don't make this translatable because it's only visible to English-language users
-				return '/month, billed annually';
-			}
-			return i18n.translate( 'per month, billed yearly' );
-		},
+		getPathSlug: () => 'premium',
+		getPriceTitle: () => i18n.translate( '$99 per year' ), //TODO: DO NOT USE
```

Comment:
Please don't make something translatable that is not meant to be translated. It's also a bad idea to hardcode dollar values into a translation. Please either use a variable or not make it translatable at all.